skins/oasis/modules/templates/Body_Index.php

```diff
@@ -32,12 +35,13 @@
 <section id="WikiaPage" class="WikiaPage<?= empty( $wg->OasisNavV2 ) ? '' : ' V2' ?><?= !empty( $isGridLayoutEnabled ) ? ' WikiaGrid' : '' ?>">
 	<div id="WikiaPageBackground" class="WikiaPageBackground"></div>
 	<div class="WikiaPageContentWrapper">
-		<?= $app->renderView( 'BannerNotifications', 'Confirmation' ) ?>
 
 		<?php if ( !empty( $wg->EnablePremiumPageHeader ) && empty( $wg->SuppressWikiHeader ) ): ?>
 			<div class="PremiumPageHeader">
 				<?= $app->renderView( 'PremiumPageHeader', 'wikiHeader' ) ?>
 			</div>
+		<?php else: ?>
+			<?= $app->renderView( 'BannerNotifications', 'Confirmation' ) ?>
```

Comment:
For new header purposes, I've moved it to line 23, and this is the `old` behavior for not tested wikis.